tests/functional/export.bats

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env bats -x
+load common
+load helpers
+load env_setup
+
+FIXTURES="$FIXTURES/export"
+CUSTOM_WORKDIR="$FIXTURES/workdir"
+WORKDIR="$FIXTURES/.lago"
+
+REPO_STORE="$FIXTURES"/repo_store
+STORE="$FIXTURES"/store
+REPO_CONF="$FIXTURES"/template_repo.json
+REPO_NAME="local_tests_repo"
+VM_NAME=lago_functional_tests_vm01
+
+SA_ENV="sa_exported_env"
+LAYERED_ENV="layered_exported_env"
+STAND_ALONE_EXPORT_DIR="${FIXTURES}/${SA_ENV}"
+LAYERED_EXPORT_DIR="${FIXTURES}/${LAYERED_ENV}"
+
+# needed in order to run libguestfs inside mock
+export LIBGUESTFS_BACKEND=direct
+
+@test "export.init" {
+    local workdir="$FIXTURES"/workdir
+    local suite="$FIXTURES"/suite.json
+
+    rm -rf "$WORKDIR" "$STORE"
+
+    # This is needed to be able to run inside mock, as it uses some temp files
+    # and that is not seamlesly reachable from out of the chroot by
+    # libvirt/kvm
+    export BATS_TMPDIR BATS_TEST_DIRNAME
+    export LIBGUESTFS_DEBUG=1 LIBGUESTFS_TRACE=1
+    cd "$FIXTURES"
+
+    helpers.run_ok "$LAGOCLI" \
+        init \
+        --template-repo-path "$REPO_CONF" \
+        --template-repo-name "$REPO_NAME" \
+        --template-store "$STORE" \
+        "$suite"
+
+    # This is needed to be able to run sudo inside the chroot
+    echo 'Defaults:root !requiretty' > /etc/sudoers.d/lago_functional_tests
+}
+
+@test "export.start: start everything at once" {
+    common.is_initialized "$WORKDIR" || skip "workdir not initiated"
+    cd "$FIXTURES"
+    helpers.run_ok "$LAGOCLI" start
+}
+
+@test "export.place_dummy_file" {
+    cd $FIXTURES
+    rm -rf dummy_file
+
+    local content="$(date)"
+    echo "$content" > "dummy_file"
+
+    helpers.run_ok "$LAGOCLI" \
+        copy-to-vm \
+        "$VM_NAME" \
+        dummy_file \
+        /root/dummy_file_inside
+
+    # Just to be sure that the file was written to disk
+    sleep 2
+}
+
+@test "export.fail_to_export_running_env" {
+    cd "$FIXTURES"
+
+    helpers.run_nook "$LAGOCLI" \
+        "export" \
+        --standalone \
+        --dst-dir "$STAND_ALONE_EXPORT_DIR"
+}
+
+@test "export.stop" {
+    cd "$FIXTURES"
+
+    "$LAGOCLI" shell "$VM_NAME" "cd /sbin && ./poweroff"
```

Comment:
why is this needed before export? and if so - it should be documented in the CLI.